documentation/source/PhaseCallbacks.md

```diff
@@ -0,0 +1,280 @@
+# Phase Callbacks in SG
+
+Integrating your own code into an already existing training pipeline can draw much effort on the user's end.
+To tackle this challenge, a list of callables triggered at specific points of the training code can be passed through `phase_calbacks_list` inside `training_params` when calling `Trainer.train(...)`.
+
+SG's `super_gradients.training.utils.callbacks` module implements some common use cases as callbacks:
+
+    ModelConversionCheckCallback
+    DeciLabUploadCallback
```

Comment:
please remove DeciLabUploadCallback. it is only for "pro" users

Reply:
fixed

Reply:
fixed in latest commit.